### ISSUE TITLE
Function argument destructuring doesn't work

### ISSUE THREAD

**Issue by ikabirov** [JSMonk/hegel#98]
https://jsmonk.github.io/hegel/try#GYVwdgxgLglg9mABHA1gCgIYCcDmBnASkQG8AoASG3wDoNSBfU00SWBRYDGAGzzWIz0iZSgyA

**Comment by JSMonk** [comment]
Yes. Currently, we don't implement a desctruction syntax, but it will be fixed soon.
Thank you for your issue ^_^.
